web_ui/src/core/labels/label.interface.ts

```diff
@@ -56,6 +57,8 @@ export enum LABEL_BEHAVIOUR {
     // Used to guarantee that when an anomalous annotation is added, its roi is
     // also given a global anomalous label
     ANOMALOUS = 1 << 4,
+
+    BACKGROUND = 1 << 5,
```

Comment:
since we're adding `BACKGROUND`, should we consider (on a different PR perhaps) adding `EMPTY` as well?

Reply:
> since we're adding `BACKGROUND`, should we consider (on a different PR perhaps) adding `EMPTY` as well?

That behaviour is handled via `EXCLUSIVE`, when a label has this behaviour the annotator won't allow other labels at the same global scope.
So for instance for all local tasks there can be only 1 empty global annotation, but if you have detection -> classification, then at the classification level there can be multiple detection annotations that have an empty classification label.